test/controllers/DraftMetadataUploadControllerSpec.scala

```diff
@@ -61,18 +73,62 @@ class DraftMetadataUploadControllerSpec extends FrontEndTestHelper {
       contentType(draftMetadataUploadPage) mustBe Some("text/html")
       pageAsString must include("<title>Page not found - Transfer Digital Records - GOV.UK</title>")
     }
+
+    "when upload successful must redirect to checks page" in {
```

Comment:
Maybe rename test to: `redirect to draft metadata checks page when upload successful`

Reply:
@ian-hoyle Rename the test as per previous comment

Reply:
renamed to redirect to draft metadata checks page when upload successful